.gitignore

```diff
@@ -1 +1,2 @@
 /Build
+/Tests/UnitTests/dist/
```

Comment:
I don't know if this is from your PR, but we typically avoid generating files in the source tree. Generated files should go in the CMake binary folder which we typically put in `Build`.